engine.vcxproj

```diff
@@ -759,6 +759,7 @@
     </ClCompile>
     <ClCompile Include="framework\Session.cpp" />
     <ClCompile Include="framework\Session_menu.cpp" />
+    <ClCompile Include="framework\Tracing.cpp" />

```

Comment:
It is quite interesting whether it should be in idlib or the main project.

Putting it into the main project means we cannot have traces in idlib code. Might become a problem in theory. On the other hand, we can easily change it later.